recipe/patch_yaml/zlib.yaml

```diff
@@ -1,9 +1,18 @@
 # zlib was pinning to minor level, but 1.2 is compatible with 1.3
 if:
   has_depends: libzlib >=1.2*
-  timestamp_gt: 1714521600000  # 2024-05-01 00:00 UTC
+  timestamp_ge: 1714521600000  # 2024-05-01 00:00 UTC
```

Comment:
If we want, we could lower the lower bound here, rather than adding a new section. Not sure what people prefer, for now I've kept it separate, to give each PR a section.